src/client/components/MyProfile/ProfilePage/ProfileComponent.js

```diff
@@ -0,0 +1,80 @@
+import { useAuthentication } from '../../../hooks/useAuthentication';
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import './ProfileComponent.styles.css';
+import { ProfileCardComponent } from '../../ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../YourUploadedCVs/YourUploadedCVs';
+import TitleDesc from '../../Title/TitleDesc';
+import { SentReviewsComponent } from '../../SentReviewsComponent/SentReviewsComponent.js';
+
+export default function ProfileComponent() {
+  const { userData } = useAuthentication();
+  const history = useHistory();
+  const [cvsList, setCvsList] = useState([]);
+  const [reviews, setReviews] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [reviewsLoaded, setReviewsLoaded] = useState(false);
+  useEffect(() => {
+    if (userData) {
+      (async () => {
+        try {
+          const response = await fetch('/api/cvs');
+          if (response.status !== 200) {
+            throw new Error('fail to connect to the api/cvs');
+          }
+          const userCvs = await response.json();
+          setIsLoaded(true);
+          if (userCvs) {
+            setCvsList(() =>
+              userCvs
+                .filter((cv) => cv.fk_user_id === userData.id)
+                .map((cv) => {
+                  return { ...cv, createdDate: cv.createdAt };
+                }),
+            );
+          }
+          const responseRew = await fetch(`api/reviews/${userData.id}`);
+          if (responseRew.status !== 200) {
+            throw new Error('fail to connect to the api/reviews');
+          }
+          const reviewsCv = await responseRew.json();
+          setReviewsLoaded(true);
+          if (reviewsCv) {
+            setReviews(reviewsCv);
+          }
+        } catch (error) {
```

Comment:
this hook is becoming quite big and I would have considered moving it out as a separate function and I would also consider splitting it into two hooks instead of one. 

One to fetch the `cvs` e.g. called `useCVs`
The second one to fetch the users reviews e.g. called `useUserReviews`

It is fine like this but would make it a lot more reusable and readable.